controller.py

```diff
@@ -826,7 +826,7 @@ def controller(
                 clogger.debug(f"Process time running mean: {p_time_mean:.3f}s")
                 # Remove process from list
                 p_list.pop(i)
-                break
+                break  # TODO AW: why break here and not iterate over all processes?
```

Comment:
I guess the idea was to immediately schedule a new task once a process is freed up. I am not sure though if that makes sense. I would also iterate over all processes here.

Reply:
:+1: Sounds good, just fixed and tested it with the example.